MapboxNavigation/NavigationMapView.swift

```diff
@@ -65,6 +73,24 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     let alternateSourceIdentifier = "alternateSource"
     let alternateLayerIdentifier = "alternateLayer"
     
+    /**
+     Attribute name for the route line that is used for identifying whether a RouteLeg is the current active leg.
+     */
+    public let currentLegAttribute = "isCurrentLeg"
+    
+    /**
+     Attribute name for the route line that is used for identifying different `CongestionLevel` along the route.
+     */
+    public let congestionAttribute = "congestion"
+
+    let routeLineWidthAtZoomLevels: [Int: MGLStyleValue<NSNumber>] = [
```

Comment:
This variable along with `congestionAttribute` and `currentLegAttribute` got moved out to Constants.swift.